features/media-queries-prefers-contrast.yml

```
@@ -0,0 +1,6 @@
+name: prefers-contrast
+description: The `prefers-contrast` CSS media feature detects the requested contrast, more or less.
```

```suggestion
description: The `prefers-contrast` CSS media feature sets styles based on whether the user prefers more or less contrast.
```

I like that all previous features use the "sets styles based on" phrase. So I'm changing it here for consistency.